README.md

```diff
@@ -305,8 +307,8 @@ $ npm test
 ## Acknowledgements
 
 This project is kindly sponsored by:
-- [nearForm](http://nearform.com)
-- [LetzDoIt](http://www.letzdoitapp.com/)
+- [nearForm](https://nearform.com)
```

Comment:
Since `https://nearform.com` will return `301 Move Permanently`. Use `https://www.nearform.com` will load faster.
```suggestion
- [nearForm](https://www.nearform.com)
```

Reply:
thanks!